libs/labelbox/src/labelbox/schema/export_task.py

```diff
@@ -106,33 +106,15 @@ class JsonConverterOutput:
 
 
 class JsonConverter(Converter[JsonConverterOutput]):  # pylint: disable=too-few-public-methods
-    """Converts JSON data."""
+    """Converts NDJSON data."""
 
     def _find_json_object_offsets(self, data: str) -> List[Tuple[int, int]]:
         object_offsets: List[Tuple[int, int]] = []
-        stack = []
-        current_object_start = None
-
-        for index, char in enumerate(data):
-            if char == "{":
-                stack.append(char)
-                if len(stack) == 1:
-                    current_object_start = index
-                    # we need to account for scenarios where data lands in the middle of an object
-                    # and the object is not the last one in the data
-                    if index > 0 and data[index -
-                                          1] == "\n" and not object_offsets:
-                        object_offsets.append((0, index - 1))
-            elif char == "}" and stack:
-                stack.pop()
-                # this covers cases where the last object is either followed by a newline or
-                # it is missing
-                if len(stack) == 0 and (len(data) == index + 1 or
-                                        data[index + 1] == "\n"
-                                       ) and current_object_start is not None:
-                    object_offsets.append((current_object_start, index + 1))
-                    current_object_start = None
-
+        split_data = data.strip().split("\n")
```

Comment:
Won't the `strip` change the offsets ? Because we're also doing ` json_str=raw_data[offset_start:offset_end + 1]` on the raw_data on line 134

Reply:
which line are you referring to specifically? 134 was removed

Reply:
sorry, I meant 134 new line number after this change. 
```
yield JsonConverterOutput(
                current_offset=current_offset + offset_start,
                current_line=current_line + line,
                json_str=raw_data[offset_start:offset_end + 1].strip(),
            )
```

Reply:
in the `convert` method